client/src/offline-settings/settings.tsx

```diff
@@ -1,76 +1,68 @@
 import { Switch } from "../ui/atoms/switch";
-import { SettingsData, STATE, UpdateStatus } from "../app-interface";
+import { SettingsData, STATE, UpdateStatus } from "./mdn-worker";
 import useInterval from "@use-it/interval";
 
-import "./index.scss";
 import { useEffect, useState } from "react";
 import UpdateButton from "./update";
 import ClearButton from "./clear";
-import { MDN_APP_DESKTOP } from "../constants";
+
+function displayEstimate({ usage = 0, quota = Infinity }: StorageEstimate) {
+  const usageInMib = Math.round(usage / (1024 * 1024));
+
+  return `${usageInMib} MiB`;
+}
 
 export default function SettingsApp({ ...appProps }) {
   return (
-    <form>
-      <div className="field-group">
-        <section>
-          <h3>Offline settings</h3>
-
-          <Settings />
-        </section>
-      </div>
-    </form>
+    <section className="field-group">
+      {/* <h3>Offline settings</h3> */}
+      <Settings />
+    </section>
   );
 }
 
 function Settings() {
-  MDN_APP_DESKTOP && window.Desktop.setTitle("Settings");
+  document.title = "MDN - Settings";
```

Comment:
I believe the standard pattern is different:

```suggestion
  document.title = "Settings | MDN";
```